src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -73,11 +73,21 @@ const useFagsakApi = (
             return;
         }
 
+        if (context.begrunnelse.valideringsstatus == Valideringsstatus.IKKE_VALIDERT) {
```

Comment:
Validate on change of the field. And then show errors on submit. This is the way the other forms work and I think it is nice that it is consistent. We may change it to display errors onBlur in the future.

Reply:
ok